patent.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import requests
+import json
+
+
+def patent(**kwargs):
+    patent_url = 'https://api.nasa.gov/patents/content'
+    get_params = {}
+    for key, value in kwargs.iteritems():
+        get_params[key] = value
+    get_response = requests.get(patent_url, params = kwargs)
+
+    if get_response.status_code == 429:
```

Comment:
Great idea, but let's make sure we capture other error codes. A common approach is to throw Exceptions for anything above 2XX, but I'll leave that up to you.